test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -334,6 +334,47 @@ public void CustomNodeContainsIronPythonDependencyTest()
             DispatcherUtil.DoEvents();
         }
 
+        /// <summary>
+        /// This test verifies that the IronPython dialog wont show the second time a custom node is opened
+        /// even if it contains IronPython nodes
+        /// </summary>
+        [Test]
+        public void WillNotDisplayDialogWhenOpeningCustomNodeWithIronPythonNodesSecondTimeInSameSession()
+        {
+            DebugModes.LoadDebugModesStatusFromConfig(Path.Combine(GetTestDirectory(ExecutingDirectory), "DynamoCoreWpfTests", "python2ObsoleteMode.config"));
```

Comment:
maybe your test should disable this config mode before the next test?

Reply:
I guess each test start its own instance of the sandbox so this should not be a problem, also, it is not disabled for any other test, before it as well which are passing.

Reply:
Each test does not start it's own instance of sandbox - they run in two different scenarios.

on master-15 all tests run in a single process.
on master-5 there are some number of parallel processes with multiple tests inside, which tests is not deterministic.

Reply:
ok, got it.

Reply:
The problem was with the casting of CurrentWS into CustomNodeWS. Still have to figure out the other 2 failing tests, looking into those.

Reply:
All test pass now